server/sonar-server/src/main/resources/org/sonar/server/branch/pr/ws/list-example.json

```diff
@@ -1,7 +1,7 @@
 {
   "pullRequests": [
     {
-      "id": "feature/bar",
+      "id": "123",
       "title": "feature/bar",
```

Comment:
Shouldn't you change the sample title to make it clear this is maybe not equal to the branch name?

Reply:
Good idea, thanks!